src/containers/unrolledlist.d

```diff
@@ -358,9 +358,9 @@ struct UnrolledList(T, Allocator = Mallocator,
 			}
 		}
 
-		ET front() const @property @trusted @nogc
+		ref ET front() const @property @trusted @nogc
 		{
-			return cast(T) current.items[index];
+			return *(cast(T*) &current.items[index]);
```

Comment:
Change this to `return *(cast(ET*) &current.items[index]);` and the tests will pass.